scripts/wait-for-services.sh

```diff
@@ -4,9 +4,9 @@ set -e
 
 cmd="$@"
 
-if [ ! -z "$POSTGRESQL_DATABASE" ]; then
+if [ ! -z "$DB_HOST" ]; then
   >&2 echo "Checking if PostgreSQL is up"
-  until PGPASSWORD="$POSTGRESQL_PASSWORD" psql -h "$POSTGRESQL_HOST" -p "$POSTGRESQL_PORT" -U "$POSTGRESQL_USER" -d "$POSTGRESQL_DATABASE" -c '\q' -q 2>/dev/null; do
```

Comment:
It's broken. On `docker-compose up --build db platform`, I've got:
~~~
....
platform_1  | JSON generated: 29/30
platform_1  | JSON generated: 30/30
platform_1  | + ./wait-for-services.sh ./send-kafka-requests.sh
platform_1  | Checking if PostgreSQL is up
platform_1  | PostgreSQL is unavailable - sleeping
platform_1  | PostgreSQL is unavailable - sleeping
...
~~~

Reply:
Fixed.